themes/default/content/docs/intro/concepts/resources/options/retainOnDelete.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "RetainOnDelete"
+meta_desc: The `retainOnDelete` resource option marks a resource to be retained during a delete operation.
+menu:
+  intro:
+    identifier: retainOnDelete
+    parent: options
+    weight: 13
+---
+
+The `retainOnDelete` resource option marks a resource to be retained. If a resource is retained then when
+Pulumi deletes or replaces the resource it will not call through to the resource provider's `Delete` method.
```

Comment:
Nit: I think we can make this sentence a bit more clear. Something like "If this option is set, then Pulumi will not call through to the resource provider's Delete method during `pulumi destroy` operation.".